syntax = "proto3";

package ananas.rpc.test;

option cc_generic_services = true;

// sample
message EchoRequest {
    string text = 1;
}
message EchoResponse {
    string text = 1;
}

service UpperService { 
    rpc ToUpper(EchoRequest) returns(EchoResponse) {
    }  
}

service AppendDotsService { 
    rpc AppendDots(EchoRequest) returns(EchoResponse) {
    }  
}

